src/redux/books/books.js

```diff
@@ -1,19 +1,18 @@
 const BOOK_ADDED = 'BOOK_ADDED';
 const BOOK_REMOVED = 'BOOK_REMOVED';
 
-let nextBook = 0
-const addBook = author => ({
+const nextBook = 0;
```

Comment:
- [OPTIONAL ] Let's set the initial state with an empty array above eg: `const initialState = [ ];`